keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/plugin/ResourceHandler.kt

```diff
@@ -173,7 +183,10 @@ fun Collection<ResourceHandler<*, *>>.supporting(
   apiVersion: ApiVersion,
   kind: String
 ): ResourceHandler<*, *> =
-  find { it.apiVersion == apiVersion && it.supportedKind.first.singular == kind }
+  find {
+    it.apiVersion == apiVersion &&
+      (it.supportedKind.first.singular == kind || it.supportedKind.first.plural == kind)
```

Comment:
Curious why this change is necessary

Reply:
Clouddriver types are generally plural but I don't know if this is true for all cloud providers.